docs/docs/examples/guides/ur-harware-interface.md

```diff
@@ -0,0 +1,577 @@
+            transitions:
+              on_load:
+                switch_controllers:
+                  hardware: hardware
+                  activate: robot_state_broadcaster
+        joint_trajectory_controller:
+          plugin: aica_core_controllers/trajectory/JointTrajectoryController
+          events:
+            predicates:
+              has_trajectory_succeeded:
+                application: stop
+            transitions:
+              on_load:
+                switch_controllers:
+                  hardware: hardware
+                  activate: joint_trajectory_controller
+  graph:
+    positions:
+      buttons:
+        button:
+          x: -460
+          y: 600
+      hardware:
+        hardware:
+          x: 620
+          y: -20
+      sequences:
+        sequence:
+          x: 40
+          y: 560
+    buttons:
+      button:
+        on_click:
+          sequence:
+            start: sequence
+    edges:
+      sequence_sequence_event_trigger_2_hardware_hardware_joint_trajectory_controller_set_trajectory:
+        path:
+          - x: 420
+            y: 1060
+          - x: 620
+            y: 1060
+          - x: 620
+            y: 900
+      sequence_sequence_event_trigger_1_hardware_hardware_joint_trajectory_controller_set_trajectory:
+        path:
+          - x: 240
+            y: 860
+      hardware_hardware_joint_trajectory_controller_has_trajectory_succeeded_on_stop_on_stop:
+        path:
+          - x: 540
+            y: 780
+          - x: 540
+            y: 480
+          - x: -20
+            y: 480
+          - x: -20
+            y: 140
+
+  ```
+</details>
+
+### Run an AICA application as one node of a program
+
+This second case requires the [External Control URCap](https://github.com/UniversalRobots/Universal_Robots_ROS2_Driver/tree/jazzy/ur_robot_driver/resources) to be installed. This is the easiest way to get started and test the
+integration with AICA functionality, as it allows users to keep their workflows intact, and only hand over control to an
+AICA application in a controlled manner and whenever that is required. After the application completes its task,
+control may be hand over back to the main node. To do that, follow the next steps:
+
+1. The external control URCap needs to be configured to the right remote control address. Navigate to the **Installation** tab
+and set the address to the one of the device that will be running the AICA application.
+
+<div class="text--center">
+  <img src={urHWINetworkingSettings} alt="External control URCap networking settings" />
+</div>
+
+2. The robot should remain in local mode, and the external control node placed within the program in the desired location.
+
+<div class="text--center">
+  <img src={urHWIURProgram} alt="UR Program with external control URCap" style={{ width: '40%' }} />
+</div>
+
+3. Headless mode in the hardware interface should be set to false, and no controller should be activated when the program starts. 
```

Comment:
feel like here we should emphasize by making this bold, although it is quite clearly a separate step

from future Vaios: this is actually done in the next section step 3